examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -425,6 +425,24 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
         }
       });
     }
+    
+    //For randomize coordinates by bounding_box
+    var gseed;
+
+    function CustomRandom() {
```

Comment:
Let's put a simple comment for each function.

Reply:
Done